api/src/Rbac/Storage/DbStorage.php

```diff
@@ -0,0 +1,255 @@
+<?php
+
+/*
+ * This file is part of the Spira framework.
+ *
+ * @link https://github.com/spira/spira
+ *
+ * For the full copyright and license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace Spira\Rbac\Storage;
+
+use Illuminate\Database\ConnectionInterface;
+use Illuminate\Database\ConnectionResolverInterface;
+use Spira\Rbac\Item\Assignment;
+use Spira\Rbac\Item\Item;
+use Spira\Rbac\Item\Permission;
+use Spira\Rbac\Item\Role;
+
+class DbStorage implements StorageInterface
+{
+    /**
+     * @var ConnectionResolverInterface
+     */
+    private $connection;
+
+    public function __construct(ConnectionResolverInterface $connection)
+    {
+        $this->connection = $connection;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getAssignments($userId)
+    {
+        if (empty($userId)) {
+            return [];
+        }
+
+        $data = $this->getConnection()->select(
+            'select * from auth_assignment WHERE user_id = ?', [$userId]
```

Comment:
Please use fluent to compose queries http://laravel.com/docs/5.1/queries#selects